libraries/testing/contracts/sysio.roa/sysio.roa.cpp

```diff
@@ -3,6 +3,45 @@
 
 namespace sysio {
 
+    bool is_sysio_account(const name& account) {
+        return account.prefix() == "sysio"_n;
+    }
+
+    roa::resources_t roa::update_reslimit(const name& owner, const asset& netWeight, const asset& cpuWeight, int64_t ram_bytes) {
```

Comment:
I think this function should be named `increase_reslimit` to better match it's semantics.  When reading `update` I tend to assume the passed in arguments are absolute target values, not incremental increases.   I also usually expect an `update_` function to be idempotent, which this is definitely not.

I also wonder whether we want a separate `set_reslimit` function for the cases when we don't want to increment. Initial roa setup might be a good case for this, as it makes me a bit uncomfortable to think we might double-initialize in certain odd scenarios.

Reply:
https://github.com/Wire-Network/wire-sysio/pull/46/commits/321593acee748039a3bdf3af0edfaf81d55e9f4e